syntax = "proto3";
package netcommsvc;


service NetCommProvider {
  rpc Send(stream SendReq) returns (stream SendReply) {}
}

message SendReq {
  string task_id = 1;
  string party_id = 2;
  bytes content = 3;
}

message SendReply {
  bool ok = 1;
}
